src/main/webapp/resources/angular/control/controlDonateCtrl.js

```diff
@@ -43,15 +43,24 @@
 
         };
 
-        self.saveDonate = function (donate, id) {
+        self.saveDonate = function (changeDonate, donate) {
             var donation = {
-                "id": id,
-                "donate": donate
+                "id": donate.id,
+                "donate": changeDonate
             };
-            ControlDonatesService.saveOwnDonate(donation).then(function (result) {
-                self.donates = result.data;
-                Notification({message: 'Donation updated successful', title: 'Notification'}, 'success');
-            })
+            if (ControlDonatesService.isDonateHaveProjectWithLess90PercentDonate(donate.percentDonateProject) ||
+                ControlDonatesService.isIncreaseDonation(changeDonate, donate.donateAmount)) {
```

Comment:
Merge these two methods into single one that will express what you are checking this it.